src/brain.coffee

```diff
@@ -789,6 +789,9 @@ class Brain
     reply = reply.replace(/\\n/ig, "\n")
     reply = reply.replace(/\\#/ig, "#")
 
+    # Prevent empty redirects
+    reply = reply.replace(/\{@\}/ig, "{@*}")
```

Comment:
@kirsle i went for the simple and dirtiest fix., but can confirm that it works both with and without utf8 mode. 

trying out changing the match regexp is good, but it does mean that the wildcard will return "undefined" if it is called from a <get> or <bot>, and "" if <star> was blank.